package.json

```diff
@@ -41,7 +41,6 @@
 		"type-fest": "^0.4.1",
 		"webext-domain-permission-toggle": "^0.1.0",
 		"webext-dynamic-content-scripts": "^5.0.2-0",
-		"webext-options-sync": "^0.15.5",
```

Comment:
Uhm?

Reply:
I'm having a bad day manually editing `package-lock.json` file to not include updates to other packages. Because now we have new JS linting error because `eslint-config-xo-typescript` auto updates inside lock file even when I'm only updating `stylelint-config-xo`.